codegen/src/main/twirl/templates/ScalaClient/Client.scala.txt

```diff
@@ -96,13 +96,13 @@ final class @{service.name}Client(settings: GrpcClientSettings)(implicit mat: Ma
   @if(methodType == akka.grpc.gen.BidiStreaming) {MethodDescriptor.MethodType.BIDI_STREAMING }
 }
 
-object @{service.name}Client {
+object @{service.name}Client { clientObj =>
```

Comment:
Ah, looks like we don't need the `clientObj` here anymore either, missed that in my PR, sorry

Reply:
I've removed `clientObj` now.